include/OsmAndCore/Map/SqliteHeightmapTileProvider.h

```diff
@@ -0,0 +1,55 @@
+#ifndef _OSMAND_CORE_SQLITE_HEIGHTMAP_TILE_PROVIDER_H_
+#define _OSMAND_CORE_SQLITE_HEIGHTMAP_TILE_PROVIDER_H_
+
+#include <OsmAndCore/stdlib_common.h>
+
+#include <OsmAndCore/QtExtensions.h>
+#include <QDir>
+#include <QString>
+
+#include <OsmAndCore.h>
+#include <OsmAndCore/CommonTypes.h>
+#include <OsmAndCore/PrivateImplementation.h>
+#include <OsmAndCore/Map/IMapElevationDataProvider.h>
+
+namespace OsmAnd
+{
+    class ITileSqliteDatabasesCollection;
+
+    class SqliteHeightmapTileProvider_P;
+    class OSMAND_CORE_API SqliteHeightmapTileProvider : public IMapElevationDataProvider
+    {
+        Q_DISABLE_COPY_AND_MOVE(SqliteHeightmapTileProvider);
+
+    private:
+        PrivateImplementation<SqliteHeightmapTileProvider_P> _p;
+    protected:
+    public:
+        SqliteHeightmapTileProvider(
+            const std::shared_ptr<const ITileSqliteDatabasesCollection>& sourcesCollection,
+            uint32_t outputTileSize
+        );
+        virtual ~SqliteHeightmapTileProvider();
+
+        const std::shared_ptr<const ITileSqliteDatabasesCollection> sourcesCollection;
+        const uint32_t outputTileSize;
+
+        virtual ZoomLevel getMinZoom() const;
```

Comment:
`Q_DECL_OVERRIDE` is missed here, and for other overridden methods too.
The absence of this keyword will generate a lot of annoying warnings during the project build process.

Reply:
Thanks, good catch.